RepClient/src/app/home/home.page.scss

```diff
@@ -1,4 +1,4 @@
-.welcome-card img {
-  max-height: 35vh;
-  overflow: hidden;
+ion-spinner {
+  width: 100%;
+  transform: scale(3);
```

Comment:
Seen you use this styling previously (e.g. company.page.scss). Consider using a universal style sheet

Reply:
No because i do not want all spinners in all page to appear like this. This is only for specific full length wide items. There is a further plan to make a component for the loading as that is the same code as well thus transferring the style to only one place.